src/FSCache.js

```diff
@@ -70,7 +71,9 @@ class FSCache {
     try {
       await fs.unlink(this.getCacheFile(filename));
       this.invalidated.delete(filename);
-    } catch (err) {}
+    } catch (err) {
```

Comment:
Did ESLint complain about this?

Also, never really a good idea to have anything fail silently, imo. But that's another discussion.

Reply:
Yes, that's why I added a comment: I did not want to deactivate this rule